services/management/ia/deps.go

```diff
@@ -22,12 +22,16 @@ import (
 	"github.com/percona/pmm/api/alertmanager/ammodels"
 )
 
+//go:generate mockery -name=alertManager -case=snake -inpkg -testonly
+//go:generate mockery -name=vmAlertService -case=snake -inpkg -testonly
+
 // alertManager is is a subset of methods of alertmanager.Service used by this package.
 // We use it instead of real type for testing and to avoid dependency cycle.
 type alertManager interface {
 	GetAlerts(ctx context.Context) ([]*ammodels.GettableAlert, error)
 	Silence(ctx context.Context, id string) error
 	Unsilence(ctx context.Context, id string) error
+	RequestConfigurationUpdate()
```

Comment:
@artemgavrilov Please call this method in IA API methods where we should regenerate Alertmanager configuration (when channel is updated, etc).

Please also add unit tests for functionality you are writing now: https://codecov.io/gh/percona/pmm-managed/tree/PMM-2.0/services/management/ia Generated mocks should help there

Reply:
Ok 👌